Documentation/FAQs.md

```diff
@@ -43,3 +44,39 @@ When in doubt, please check your compiler's documentation for the options necess
 
 * #### cl.exe
   Symbols are located in the `*.pdb` file.
+
+## How to enable logging
+
+Enabling logging will show communication information between VS Code and our extension and between our extension and the debugger.
```

Comment:
This sentence seems a little vague.  Maybe be more specific on what you mean when you say "our extension" and the "debugger" just in case people don't know what's being referred to?

Reply:
Leslie and I talked offline. This is documentation in our repo so "this extension" should be clear but i'll change it to say "cpptools extension"